scripts/rollup/forks.js

```diff
@@ -158,11 +159,7 @@ const forks = Object.freeze({
   },
 
   'scheduler/src/SchedulerFeatureFlags': (bundleType, entry, dependencies) => {
-    if (
-      bundleType === FB_WWW_DEV ||
-      bundleType === FB_WWW_PROD ||
-      bundleType === FB_WWW_PROFILING
-    ) {
+    if (bundleType === FB_WWW_COMBINED) {
```

Comment:
Why did you remove the other two bundleTypes here?

Reply:
Because the purpose of the combined bundle is to replace the dev and prod bundles with a single bundle that includes both code paths